src/events/attributes.rs

```diff
@@ -59,13 +59,13 @@ pub struct Attribute<'a> {
     /// the key to uniquely define the attribute
     pub key: &'a [u8],
     /// the value
```

Comment:
Can you mention is the doc that the `value` is the `unescaped` value?

And that converting from a `(&[u8], &[u8])` doesn't try to escape the value while converting from a `(&str, &str)` does?